spark/src/main/scala/org/apache/spark/sql/delta/IdentityColumn.scala

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
This file is part of the Scala API PR right now: https://github.com/delta-io/delta/pull/2857

Reply:
Merged in https://github.com/delta-io/delta/pull/2916 to reduce conflict between PRs.